tests/src/Functional/JsonldContextGeneratorTest.php

```diff
@@ -27,6 +27,11 @@ class JsonldContextGeneratorTest extends BrowserTestBase {
     'jsonld',
   ];
 
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
```

Comment:
Currently, leaving off the theme will cause a deprecation error. It would have passed Travis without it but it kept complaining when I was running tests locally. Adding the test resolves that. I don't recall the API change notice where they talk about it.